k8s/README.md

```diff
@@ -1,9 +1,27 @@
-## Build the pipeline
+# Business Create UI
 
+## How to build the openshift environment
 
-1. **Setup parameters** Alter, or copy the _bcrs-entities-create-ui-build-pipeline.param_ updating the values for your pipeline.
-2. **Create the pipeline** process the template referencing your parameter file.
+### Images
+This repo has implemented github actions to build and push the new image into our *tools* environment in OpenShift when a PR is merged. Significance in OpenShift environment:
+- no build config (image is built already)
+- no build/dev pipeline (image pushed into dev already)
```

Comment:
This is unclear. Yes, the image is pushed to the dev project but it still needs to be tagged in order to "deploy" it. Please clarify,

Reply:
the image is built in tools and tagged to dev (or pushed to dev). I'll update it to say tagged to dev already but I think that's more confusing to people not used to dealing with images